src/test_workflow/perf_test/perf_test_cluster.py

```diff
@@ -71,3 +85,34 @@ def service(self):
 
     def dependencies(self):
         return []
+
+    def wait_for_processing(self, max_wait_sec = 60):
+        if self.public_ip is None:
+            return
+        url = "".join([self.endpoint(), "/_cluster/health"])
+        end_time = time.time() + max_wait_sec
+        while(True):
```

Comment:
It might be cleaner to use https://pypi.org/project/retry/. We already have this in our current dependencies and is being utilized here - https://github.com/opensearch-project/opensearch-build/blob/main/src/run_perf_test.py#L68

Reply:
Sure.